test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -624,6 +638,7 @@
       measurePods: 5
   - name: 500Nodes
```

Comment:
we should probably add a larger case for this.

And 18 is definitely too low, so we could be looking at it.

Reply:
cc @mengqiy

Reply:
#126858